Documentation/ReleaseNotes.md

```diff
@@ -22,6 +22,8 @@ MRTK now supports running Oculus Quest Headsets and Controllers using the native
 
 For instructions on how to deploy your device on the Oculus Quest using the new pipeline, see the [Oculus Quest Setup Guide](CrossPlatform/OculusQuestMRTK.md)
 
+There is currently a known issue for using the [Oculus XR plugin with when targetting Standalone platforms](https://forum.unity.com/threads/unable-to-start-oculus-xr-plugin.913883/).  Check the Oculus bug tracker/forums/release notes for updates.
```

Comment:
It might be worth adding this to the Known Issues section (and also, if there's a snippet of the stack that you think will be common (i.e. A Native Collection has not been disposed, resulting in a memory leak) it might be worth having that snippet there as well so that it's more searchable (or someone might see that line and remember it for later when they hit the issue, etc)